requirements/prod.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
+# This file is autogenerated by pip-compile
```

Comment:
you might need to update pip-tools to avoid some of the inconsequential changes in here